bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/ghost/GhostEditor.scala

```diff
@@ -132,19 +145,362 @@ final class GhostEditor extends ComponentEditor[ISPObsComponent, Ghost] {
     layout(asterismComboBox) = new Constraints() {
       anchor = Anchor.NorthWest
       gridx = 1
-      gridwidth = 2
       gridy = row
+      gridwidth = 2
       fill = Fill.Horizontal
+      insets = new Insets(10, 0, 0, 0)
+    }
+    row += 1
+
+
+    /*****************
+     *** Detectors ***
+     *****************/
+    object detectorUI extends GridBagPanel {
```

Comment:
Nothing like `GridBagPanel`...

Reply:
I solemnly swear to learn `MigLayout` after this is all over :D.